homeassistant/components/climate/strings.json

```diff
@@ -264,6 +284,9 @@
     "not_valid_swing_mode": {
       "message": "Swing mode {mode} is not valid. Valid swing modes are: {modes}."
     },
+    "not_valid_horizontal_swing_mode": {
```

Comment:
What about adding quotes around {mode}, I have seen cases where the mode logged was empty, something that was hard to find, at that moment just 2 spaces are printed.